server/protocols/webadmin/webadmin-jmap/src/main/java/org/apache/james/webadmin/data/jmap/dto/UserIdentity.java

```diff
@@ -0,0 +1,145 @@
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.webadmin.data.jmap.dto;
+
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.james.core.MailAddress;
+import org.apache.james.jmap.api.model.Identity;
+
+import scala.jdk.javaapi.CollectionConverters;
+import scala.jdk.javaapi.OptionConverters;
+
+public class UserIdentity {
+    public static UserIdentity from(Identity identity) {
+        return new UserIdentity(
+            identity.name(),
+            identity.email().asString(),
+            identity.id().id().toString(),
+            identity.mayDelete(),
+            identity.textSignature(),
+            identity.htmlSignature(),
+            identity.sortOrder(),
+            getBccFromIdentity(identity),
+            getReplyFromIdentity(identity));
+    }
+
+    public static class EmailAddress {
+        public static EmailAddress from(org.apache.james.jmap.api.model.EmailAddress scala) {
+            return new EmailAddress(scala.nameAsString(), scala.email());
+        }
+
+        private String emailerName;
+        private String mailAddress;
+
+        public EmailAddress(String emailerName, MailAddress mailAddress) {
+            this.emailerName = emailerName;
+            this.mailAddress = mailAddress.asString();
+        }
+
+        public String getEmailerName() {
+            return emailerName;
+        }
+
+        public String getMailAddress() {
+            return mailAddress;
+        }
+    }
+
+    private static List<EmailAddress> getBccFromIdentity(Identity identity) {
+        return OptionConverters.toJava(identity.bcc())
+            .map(CollectionConverters::asJava)
+            .orElseGet(List::of)
+            .stream()
+            .map(EmailAddress::from)
+            .collect(Collectors.toList());
+    }
+
+    private static List<EmailAddress> getReplyFromIdentity(Identity identity) {
+        return OptionConverters.toJava(identity.replyTo())
+            .map(CollectionConverters::asJava)
+            .orElseGet(List::of)
+            .stream()
+            .map(EmailAddress::from)
+            .collect(Collectors.toList());
+    }
+
+    private String name;
+    private String email;
+    private String id;
+    private Boolean mayDelete;
+    private String textSignature;
+    private String htmlSignature;
+    private Integer sortOrder;
+    private List<EmailAddress> bcc;
+    private List<EmailAddress> replyTo;
+
+    public UserIdentity(String name, String email, String id,
```

Comment:
Why you need to redeclare this object? I believe we already have one declared in james-server-data-jmap called `Identity`?

Reply:
`Identity` in data-jmap is a scala object, 
web-admin is using jackson objectmapper for serialize/deserialize, when I tried scala object, it is complex, 
So the creating a POJO java object is easier